app/models/manageiq/providers/ovirt4/infra_manager/event_parser.rb

```diff
@@ -0,0 +1,31 @@
+module ManageIQ::Providers::Ovirt4::InfraManager::EventParser
```

Comment:
so, this module not really used somewhere and I am working on this here https://github.com/ManageIQ/manageiq/pull/7104 

So, this could be injected as a dependency into the event_catcher